magento/magento2
# Inverted attribute grid system column

### Description

The Pull Request fixes small issue in product attribute grid `Catalog > Products > Product Page > Add Attribute`, where `System` column should be inverted instead displaying as is.

The `System` grid column is related with `is_user_defined` column in table, and when `is_user_defined = 0` then `System` should be `Yes` and vice versa.

### Manual testing scenario

1. Create new product attribute  - `Stores > Attributes > Product > Add New Attribute`
2. Make sure that in `Stores > Attributes > Product` grid `System` column for the newly created attribute displayed as `No`
3. Go to any backend product page and open modal attribute grid - `Catalog > Products > Product Page > Add Attribute`
4. Make sure that `System` column is also as `No` there

The fix has been covered by MFTF test.

### Resolved issues:
1. [x] resolves magento/magento2#30235: Inverted attribute grid system column
